libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2277,7 +2277,7 @@ bool AP_AHRS::attitudes_consistent(char *failure_msg, const uint8_t failure_msg_
             // check roll and pitch difference
             const float rp_diff_rad = primary_quat.roll_pitch_difference(ekf2_quat);
             if (rp_diff_rad > ATTITUDE_CHECK_THRESH_ROLL_PITCH_RAD) {
-                hal.util->snprintf(failure_msg, failure_msg_len, "EKF2 Roll/Pitch inconsistent by %d deg", (int)degrees(rp_diff_rad));
+                hal.util->snprintf(failure_msg, failure_msg_len, "EKF2 Roll/Pitch inconsistent with current AHRS. Wait or check sensors");
```

Comment:
```suggestion
                hal.util->snprintf(failure_msg, failure_msg_len, "%s Roll/Pitch inconsistent with current AHRS. Wait or check sensors", "EKF2");
```

etc elsewhere.

This will let the compiler re-use the format string, actually saving bytes on this PR.

Reply:
.... could potentially be 
```suggestion
                hal.util->snprintf(failure_msg, failure_msg_len, "%s %s inconsistent with current AHRS. Wait or check sensors", "EKF2", "Roll/Pitch");
```

Reply:
done

Reply:
Doesn't look like it was done in this PR.

This branch shows what I meant; feel free to cherry-pick the patch :-)

https://github.com/ArduPilot/ardupilot/compare/master...peterbarker:ardupilot:pr/save-bytes?expand=1

That branch saves 224 bytes vs master, whereas this one *costs* 80-odd bytes,

Reply:
okay done....credit given...thanks